### PR TITLE
Add e2e tile test

### PR DESCRIPTION
**Pull request by ivy-zhang**

### CODE REVIEW

**Comment by Cidana-Developers** [comment]
Can one of the admins verify this patch?

File: test/e2e_test/SvtAv1E2ETest.cc [hunk 1]
```diff
@@ -164,3 +168,45 @@ static const std::vector<EncTestSetting> default_enc_settings = {
...
                         ::testing::ValuesIn(default_enc_settings),
                         EncTestSetting::GetSettingName);
+
+/**
+ * @brief SVT-AV1 encoder E2E test with comparing the reconstructed frame with
+ * output frame from decoder buffer list when the tile is inverted.
+ *
+ * Test strategy:
+ * Setup SVT-AV1 encoder with different parameter, and encode the input YUV data
+ * frames. Collect the reconstructed frames and compared them with reference
```
Comment (Cidana-Developers) [inline test/e2e_test/SvtAv1E2ETest.cc]: Maybe we should pay more attention to the independence test, like why we decode the tiles in inverse order.

File: test/e2e_test/SvtAv1E2EFramework.cc [hunk 2]
```diff
@@ -226,6 +246,8 @@ void SvtAv1E2ETestFramework::init_test(TestVideoVector &test_vector) {
     // create reference decoder if required.
     if (enable_decoder) {
         refer_dec_ = create_reference_decoder(enable_analyzer);
+        if (enable_tile)
+            refer_dec_->set_invert_tile();
```
Comment (Cidana-Developers) [inline test/e2e_test/SvtAv1E2EFramework.cc]: Maybe rename the function to set_invert_tile_decoding_order to be more clear?

File: test/e2e_test/RefDecoder.cc [hunk 3]
```diff
@@ -418,3 +418,13 @@ void RefDecoder::trans_video_frame(const void* image_handle,
     frame.timestamp =
         init_timestamp_ + ((uint64_t)dec_frame_cnt_ * frame_interval_);
 }
+
+void RefDecoder::control(int ctrl_id, int arg) {
```
Comment (Cidana-Developers) [inline test/e2e_test/RefDecoder.cc:422]: Do we really need this function? Its a simple wrapper of original control function, but only called once.

**Review by Cidana-Developers — COMMENTED** [review]
The majority are good, only some minor questions.

**Comment by Cidana-Developers** [comment]
@ivy-zhang Could you check the internal decoding order in the aom decoder, I'm wondering some other parameters should be set too, like       
inv_dec_->Control(AV1_SET_DECODE_TILE_ROW, -1);
inv_dec_->Control(AV1_SET_DECODE_TILE_COL, -1);
inv_dec_->Control(AV1_SET_TILE_MODE, 0);

**Comment by ivy-zhang** [comment]
# Pr for add e2e tile test
## Check result
  1. clang-format: OK
  2. local build result:
    + windows:OK 
    + linux:
      CMake Error #1 :
 ```
      CMake Error at /opt/work/sad/SVT-AV1/test/e2e_test/test_data_util.cmake:76 (file):
  file DOWNLOAD HASH mismatch

    for file: [/opt/work/sad/SVT-AV1/test/e2e_test/../vectors//niklas_640_480_30.yuv]
      expected hash: [70894878d916a599842d9ad0dcd24e10c13e5467]
        actual hash: [214740dc3fed5dab1df5f3b19e6208f1bed4aa87]
             status: [18;"Transferred a partial file"]
```
      
  3. remote build result:
    + the branch fix_e2e_param_setup_issue haven't been merged into master, so the param I used to set up in tile test can't work correctly.
		 
## Test Plan: 
  Run the unit test with `./SvtAv1E2ETests --gtest_filter="*TILETEST*"`, and check the result
## Dry-run result:
~~~	
```
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
Note: Google Test filter = *TILETEST*
[==========] Running 4 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 4 tests from TILETEST/TileIndependenceTest
[ RUN      ] TILETEST/TileIndependenceTest.TileTest/TileTest1
EncSetting: TileCol = 0
EncSetting: TileRow = 0
Start test case TileTest1: TileCol=0, TileRow=0, test vector: kirland_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 0
EncSetting: TileRow = 0
Start test case TileTest1: TileCol=0, TileRow=0, test vector: kirland_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:116
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
Encoder EOS
PSNR: 60 frames, total: 45.5980, luma: 42.7908, cb: 46.9281, cr: 47.0751
Enc Performance: 0.00sec/frame (3157.8947FPS)
SVT: you have no memory leak
Start test case TileTest1: TileCol=0, TileRow=0, test vector: niklas_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 0
EncSetting: TileRow = 0
Start test case TileTest1: TileCol=0, TileRow=0, test vector: kirland_640_480_30.yuv
Start test case TileTest1: TileCol=0, TileRow=0, test vector: niklas_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:116
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
Encoder EOS
PSNR: 60 frames, total: 40.7054, luma: 36.1993, cb: 43.5184, cr: 42.3986
Enc Performance: 0.00sec/frame (3529.4118FPS)
SVT: you have no memory leak
[       OK ] TILETEST/TileIndependenceTest.TileTest/TileTest1 (22364 ms)
[ RUN      ] TILETEST/TileIndependenceTest.TileTest/TileTest2
EncSetting: TileCol = 0
EncSetting: TileRow = 1
Start test case TileTest2: TileCol=0, TileRow=1, test vector: kirland_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 0
EncSetting: TileRow = 1
Start test case TileTest2: TileCol=0, TileRow=1, test vector: kirland_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:118
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
Encoder EOS
PSNR: 60 frames, total: 45.6002, luma: 42.7946, cb: 46.9065, cr: 47.0995
Enc Performance: 0.00sec/frame (4000.0000FPS)
SVT: you have no memory leak
Start test case TileTest2: TileCol=0, TileRow=1, test vector: niklas_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 0
EncSetting: TileRow = 1
Start test case TileTest2: TileCol=0, TileRow=1, test vector: kirland_640_480_30.yuv
Start test case TileTest2: TileCol=0, TileRow=1, test vector: niklas_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:116
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:118
Encoder EOS
PSNR: 60 frames, total: 40.7013, luma: 36.1955, cb: 43.5178, cr: 42.3907
Enc Performance: 0.00sec/frame (3000.0000FPS)
SVT: you have no memory leak
[       OK ] TILETEST/TileIndependenceTest.TileTest/TileTest2 (22450 ms)
[ RUN      ] TILETEST/TileIndependenceTest.TileTest/TileTest3
EncSetting: TileCol = 1
EncSetting: TileRow = 0
Start test case TileTest3: TileCol=1, TileRow=0, test vector: kirland_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 1
EncSetting: TileRow = 0
Start test case TileTest3: TileCol=1, TileRow=0, test vector: kirland_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:116
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
Encoder EOS
PSNR: 60 frames, total: 45.5919, luma: 42.7680, cb: 46.9259, cr: 47.0818
Enc Performance: 0.00sec/frame (3000.0000FPS)
SVT: you have no memory leak
Start test case TileTest3: TileCol=1, TileRow=0, test vector: niklas_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 1
EncSetting: TileRow = 0
Start test case TileTest3: TileCol=1, TileRow=0, test vector: kirland_640_480_30.yuv
Start test case TileTest3: TileCol=1, TileRow=0, test vector: niklas_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:118
Encoder EOS
PSNR: 60 frames, total: 40.6852, luma: 36.1898, cb: 43.4856, cr: 42.3802
Enc Performance: 0.00sec/frame (4285.7143FPS)
SVT: you have no memory leak
[       OK ] TILETEST/TileIndependenceTest.TileTest/TileTest3 (22854 ms)
[ RUN      ] TILETEST/TileIndependenceTest.TileTest/TileTest4
EncSetting: TileCol = 1
EncSetting: TileRow = 1
Start test case TileTest4: TileCol=1, TileRow=1, test vector: kirland_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 1
EncSetting: TileRow = 1
Start test case TileTest4: TileCol=1, TileRow=1, test vector: kirland_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:118
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:116
Encoder EOS
PSNR: 60 frames, total: 45.5937, luma: 42.7747, cb: 46.9010, cr: 47.1055
Enc Performance: 0.00sec/frame (3157.8947FPS)
SVT: you have no memory leak
Start test case TileTest4: TileCol=1, TileRow=1, test vector: niklas_640_480_30.yuv
Running main() from C:\Users\Ryan\tiletest\SVT-AV1\third_party\googletest\src\gtest_main.cc
EncSetting: TileCol = 1
EncSetting: TileRow = 1
Start test case TileTest4: TileCol=1, TileRow=1, test vector: kirland_640_480_30.yuv
Start test case TileTest4: TileCol=1, TileRow=1, test vector: niklas_640_480_30.yuv
SVT [version]:  SVT-AV1 Encoder Lib v0.6.0
SVT [build]  :  Visual Studio 2017 64 bit
LIB Build date: Jul 25 2019 14:04:36
-------------------------------------------
Number of logical cores available: 12
Number of PPCS 72
-------------------------------------------
SVT [config]: Main Profile      Tier (auto)     Level (auto)
SVT [config]: EncoderMode                                                       : 8
SVT [config]: EncoderBitDepth / EncoderColorFormat / CompressedTenBitFormat     : 8 / 1 / 0
SVT [config]: SourceWidth / SourceHeight                                        : 640 / 480
SVT [config]: FrameRate / Gop Size                                              : 30 / 32
SVT [config]: HierarchicalLevels / BaseLayerSwitchMode / PredStructure          : 4 / 0 / 2
SVT [config]: BRC Mode / QP  / LookaheadDistance / SceneChange                  : CQP / 50 / 33 / 0
-------------------------------------------
SVT Memory Usage:
    total allocated memory:       1.75 GB
        malloced memory:          1.18 GB
        callocated memory:        191.01 MB
        allocated aligned memory: 396.11 MB
    mutex count: 572
    semaphore count: 344
    thread count: 75
    hash table fulless: 0.032787, hash bucket is healthy
top 10 malloced memory locations:
(227.86 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbPictureControlSet.c:1135
(144.96 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Encoder\Codec\EbEncHandle.c:3399
(143.28 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:72
(143.05 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbBitstreamUnit.c:40
(88.76 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbRestoration.c:1737
(40.78 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:60
(37.50 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbCodingUnit.c:61
(35.21 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbModeDecisionProcess.c:73
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:118
(35.16 MB): C:\Users\Ryan\tiletest\SVT-AV1\Source\Lib\Common\Codec\EbMotionEstimationContext.c:117
Encoder EOS
PSNR: 60 frames, total: 40.6905, luma: 36.1795, cb: 43.4991, cr: 42.3928
Enc Performance: 0.00sec/frame (2400.0000FPS)
SVT: you have no memory leak
[       OK ] TILETEST/TileIndependenceTest.TileTest/TileTest4 (22935 ms)
[----------] 4 tests from TILETEST/TileIndependenceTest (90615 ms total)

[----------] Global test environment tear-down
[==========] 4 tests from 1 test case ran. (90618 ms total)
[  PASSED  ] 4 tests.

~~~
```

**Review by Cidana-Developers — APPROVED** [review]

**Comment by Cidana-Developers** [comment]
Merged.

**Pull request closed without merging**